Dockerfile

```diff
@@ -24,11 +24,11 @@ FROM eclipse-temurin:21-jre AS runtime
 ARG TARGETARCH
```

Comment:
`
ARG version=1.20.4
`
Maybe also bump version to 1.20.6?

Reply:
No yet. It is still experimental. https://papermc.io/downloads/paper